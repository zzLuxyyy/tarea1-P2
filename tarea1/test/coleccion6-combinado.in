crearColeccion

crearFecha 01/10/1995
crearLibro 843397030 "Ensayo sobre la ceguera" "José" "Saramago" "Una epidemia de ceguera azota a una ciudad, revelando lo peor del ser humano" 2
agregarEnColeccion

crearFecha 16/07/1951
crearLibro 316769177 "El guardián en el centeno" "J.D." "Salinger" "Holden narra sus días tras ser expulsado del colegio, en busca de sentido" 1
agregarEnColeccion

crearFecha 01/05/1992
crearLibro 140003446 "Jazz" "Toni" "Morrison" "Una historia de amor y crimen en el Harlem de los años 20" 3
agregarEnColeccion

crearFecha 01/01/2008
crearLibro 880619315 "La soledad de los números primos" "Paolo" "Giordano" "Dos almas heridas incapaces de coincidir en el amor" 1
agregarEnColeccion

crearFecha 26/06/1997
crearLibro 171819202 "Harry Potter y la piedra filosofal" "J.K." "Rowling" "Un niño mago descubre su destino en la escuela Hogwarts" 6
agregarEnColeccion

crearFecha 01/01/2002
crearLibro 849550144 "Juventud" "J.M." "Coetzee" "Un joven sudafricano busca sentido en su vida y vocación en Londres" 4
agregarEnColeccion

crearFecha 01/01/1971
crearLibro 204578912 "Las venas abiertas de América Latina" "Eduardo" "Galeano" "Un ensayo crítico sobre la historia de la explotación en América Latina" 7
agregarEnColeccion

crearFecha 28/01/1813
crearLibro 111213141 "Orgullo y prejuicio" "Jane" "Austen" "Elizabeth Bennet enfrenta el amor y los prejuicios de su época" 1
agregarEnColeccion

crearFecha 01/01/1980
crearLibro 222324252 "El nombre de la rosa" "Umberto" "Eco" "Un fraile investiga misteriosos asesinatos en una abadía medieval" 3
agregarEnColeccion

crearFecha 01/01/1993
crearLibro 214589721 "Las aventuras del sapo Ruperto" "Roy" "Berocay" "Un simpático sapo uruguayo vive divertidas y valientes aventuras" 9
agregarEnColeccion

imprimirColeccion

removerDeColeccion 8
removerDeColeccion 10
removerDeColeccion 4

imprimirColeccion

imprimirLibrosGeneroColeccion 13

imprimirLibrosGeneroColeccion 1

liberarColeccion

Fin