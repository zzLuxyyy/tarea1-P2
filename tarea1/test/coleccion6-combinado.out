1>2>3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>18>19>20>21>22>23>24>25>26>27>28>29>30>31>32>Libro 880619315
Título: La soledad de los números primos
Autor: Paolo Giordano
Descripción: Dos almas heridas incapaces de coincidir en el amor
Género: 1
Fecha de edición: 1/1/2008
Libro 849550144
Título: Juventud
Autor: J.M. Coetzee
Descripción: Un joven sudafricano busca sentido en su vida y vocación en Londres
Género: 4
Fecha de edición: 1/1/2002
Libro 171819202
Título: Harry Potter y la piedra filosofal
Autor: J.K. Rowling
Descripción: Un niño mago descubre su destino en la escuela Hogwarts
Género: 6
Fecha de edición: 26/6/1997
Libro 843397030
Título: Ensayo sobre la ceguera
Autor: José Saramago
Descripción: Una epidemia de ceguera azota a una ciudad, revelando lo peor del ser humano
Género: 2
Fecha de edición: 1/10/1995
Libro 214589721
Título: Las aventuras del sapo Ruperto
Autor: Roy Berocay
Descripción: Un simpático sapo uruguayo vive divertidas y valientes aventuras
Género: 9
Fecha de edición: 1/1/1993
Libro 140003446
Título: Jazz
Autor: Toni Morrison
Descripción: Una historia de amor y crimen en el Harlem de los años 20
Género: 3
Fecha de edición: 1/5/1992
Libro 222324252
Título: El nombre de la rosa
Autor: Umberto Eco
Descripción: Un fraile investiga misteriosos asesinatos en una abadía medieval
Género: 3
Fecha de edición: 1/1/1980
Libro 204578912
Título: Las venas abiertas de América Latina
Autor: Eduardo Galeano
Descripción: Un ensayo crítico sobre la historia de la explotación en América Latina
Género: 7
Fecha de edición: 1/1/1971
Libro 316769177
Título: El guardián en el centeno
Autor: J.D. Salinger
Descripción: Holden narra sus días tras ser expulsado del colegio, en busca de sentido
Género: 1
Fecha de edición: 16/7/1951
Libro 111213141
Título: Orgullo y prejuicio
Autor: Jane Austen
Descripción: Elizabeth Bennet enfrenta el amor y los prejuicios de su época
Género: 1
Fecha de edición: 28/1/1813
33>No se puede remover el libro con isbn 8 porque NO pertenece a la colección.
34>No se puede remover el libro con isbn 10 porque NO pertenece a la colección.
35>No se puede remover el libro con isbn 4 porque NO pertenece a la colección.
36>Libro 880619315
Título: La soledad de los números primos
Autor: Paolo Giordano
Descripción: Dos almas heridas incapaces de coincidir en el amor
Género: 1
Fecha de edición: 1/1/2008
Libro 849550144
Título: Juventud
Autor: J.M. Coetzee
Descripción: Un joven sudafricano busca sentido en su vida y vocación en Londres
Género: 4
Fecha de edición: 1/1/2002
Libro 171819202
Título: Harry Potter y la piedra filosofal
Autor: J.K. Rowling
Descripción: Un niño mago descubre su destino en la escuela Hogwarts
Género: 6
Fecha de edición: 26/6/1997
Libro 843397030
Título: Ensayo sobre la ceguera
Autor: José Saramago
Descripción: Una epidemia de ceguera azota a una ciudad, revelando lo peor del ser humano
Género: 2
Fecha de edición: 1/10/1995
Libro 214589721
Título: Las aventuras del sapo Ruperto
Autor: Roy Berocay
Descripción: Un simpático sapo uruguayo vive divertidas y valientes aventuras
Género: 9
Fecha de edición: 1/1/1993
Libro 140003446
Título: Jazz
Autor: Toni Morrison
Descripción: Una historia de amor y crimen en el Harlem de los años 20
Género: 3
Fecha de edición: 1/5/1992
Libro 222324252
Título: El nombre de la rosa
Autor: Umberto Eco
Descripción: Un fraile investiga misteriosos asesinatos en una abadía medieval
Género: 3
Fecha de edición: 1/1/1980
Libro 204578912
Título: Las venas abiertas de América Latina
Autor: Eduardo Galeano
Descripción: Un ensayo crítico sobre la historia de la explotación en América Latina
Género: 7
Fecha de edición: 1/1/1971
Libro 316769177
Título: El guardián en el centeno
Autor: J.D. Salinger
Descripción: Holden narra sus días tras ser expulsado del colegio, en busca de sentido
Género: 1
Fecha de edición: 16/7/1951
Libro 111213141
Título: Orgullo y prejuicio
Autor: Jane Austen
Descripción: Elizabeth Bennet enfrenta el amor y los prejuicios de su época
Género: 1
Fecha de edición: 28/1/1813
37>38>Libro 880619315
Título: La soledad de los números primos
Autor: Paolo Giordano
Descripción: Dos almas heridas incapaces de coincidir en el amor
Género: 1
Fecha de edición: 1/1/2008
Libro 316769177
Título: El guardián en el centeno
Autor: J.D. Salinger
Descripción: Holden narra sus días tras ser expulsado del colegio, en busca de sentido
Género: 1
Fecha de edición: 16/7/1951
Libro 111213141
Título: Orgullo y prejuicio
Autor: Jane Austen
Descripción: Elizabeth Bennet enfrenta el amor y los prejuicios de su época
Género: 1
Fecha de edición: 28/1/1813
39>40>Fin.
